cocos/3d/lights/directional-light-component.ts

```diff
@@ -296,16 +296,16 @@ export class DirectionalLight extends Light {
         return (legacyCC.director.root as Root).pipeline.pipelineSceneData.shadows.type
         === ShadowType.ShadowMap && this._shadowFixedArea === true;
     })
-    @property({ group: { name: 'ShadowSettings', displayOrder: 13 } })
+    @property({ group: { name: 'DynamicShadowSettings', displayOrder: 13 } })
     @editable
     @type(CCFloat)
-    get fixedFar () {
+    get shadowFar () {
         return this._shadowFar;
     }
-    set fixedFar (val) {
+    set shadowFar (val) {
```

Comment:
Renaming need backward compatibility, including 

1. Declare `@formerlySerializedAs` to ensure data migration
2. Add replaceProperty in module's deprecated